src/linux/LinuxMouse.cpp

```diff
@@ -166,8 +166,8 @@ void LinuxMouse::_processXEvents()
 			//Ignore out of bounds mouse if we just warped
 			if( mWarped )
 			{
-				if(event.xmotion.x < 5 || event.xmotion.x > mState.width - 5 ||
-				   event.xmotion.y < 5 || event.xmotion.y > mState.height - 5)
+				if(event.xmotion.x < 100 || event.xmotion.x > mState.width - 100 ||
```

Comment:
Does this refer to what you where having issues about the window border and screen border having the same width?
100 seems a bit high to be honest... I need to do some testing.

Reply:
If I recall, the issue was with certain desktop envoriments that have high margins. Such as the side panel on Ubuntu's Unity. My memory on this is fuzzy since the PR was only submitted by an IRC request several years ago. If you have a program running Windowed almost at the size of the monitor, with something like Unity the mouse in the program will become offset if it warps due to the exclusive margins in the DE. In my case, cranking it up to 100 fixed the issue on all major DEs I tested it on. Current code is distrbuted to about 2,000 linux users and I have not had a complaint about it since.   

Sorry for the messiness, looks like I botched this PR from my SVN. I can't stand GIT ;)

Reply:
(I can't stand SVN :wink: )

I think I understand the problem you had. I will try to see if I can replicate that particular issue, but I'm unsure about that. In any case, I will make theses two margin threshold higher, it shouldn't hurt. 

You may have guessed, but I just "joined" the project here. I am trying to get the development of OIS back on track, starting by merging the community provided fixes and improvement back into this repo that is now the official upstream one. It would in the end simplify things for you as you will not have to maintain your own fork of OIS, and I'm planning to get the thing working with other "backend" (I need to validate if the mac os x/Cocoa code is actually working, and I'm looking at ditching DirectInput for Xinput, and to make the library compatible with the Windows Universal Platform, and if possible, see if we can support Wayland on Linux without some black magic involving Wayland emulating Xorg) 

I think that OIS still can be useful for people, and I know that there's legacy code that uses it out there. 

Do you have some kind of beta program with your existing Linux users and your particular application ? Would it be envisage-able for you to help us implement the fixes you have made on your own fork of the project, and you will benefit from the planned improvements. 

I will open a specific issue to discuss theses things ;-)

Reply:
Sorry for the late reply.  Crunch time an update. 

There isn't too much different from my code than what is on here. I do have a "testing" build for users, however it's fairly stable. In the coming months I do plan on doing an engine update, I might be able to have some user testing then. Outside of OSX, there wasn't much plans for me to touch the input systems during engine updating. But we'll see depending on how much your efforts progresses the project. :)

Reply:
The idea is not to break library compatibility. (The Linux keyboard support is already a bit nicer than it was before, and it un-breaks some weirdness people that where not using english keyboard could experience). I would argue that in any case if you are upgrading "some libraries" you use, you should also upgrade this one if we have made a new release ;-)

We technically have, but it hasn't been tagged as one: if you are running something that is called "ios 1.3" it's really old, like 2010 old. 

At some point, the version number has been bumped to 1.4, but I don't have any good history on this. I'm planing on attempting to tag version numbers to the commit history. See issue #26. But I have to figure out how to go about it...)